jobs/base_jobs.py

```diff
@@ -62,21 +62,35 @@
 from jobs import job_options
 import python_utils
 
-import apache_beam as beam
-from apache_beam import runners
 
-
-class JobMetaClass(type):
-    """Meta class for all of Oppia's Apache Beam jobs.
+class JobMetaclass(type):
+    """Metaclass for all of Oppia's Apache Beam jobs.
 
     This class keeps track of the complete list of jobs. The list can be read
-    with the JobMetaClass.get_jobs() class method.
+    with the JobMetaclass.get_jobs() class method.
```

Comment:
Can you explain here in more detail why we are using a metaclass? We typically avoid those in our regular development since they introduce complications (see e.g. first paragraph [here](https://blog.ionelmc.ro/2015/02/09/understanding-python-metaclasses/)) so it might be worth explaining why this one is worth it. (For example, it might be because this is the standard way to do Apache Beam stuff and is referenced in the official Beam docs at [link].)